app/views/shared/_status_panel.html.erb

```diff
@@ -0,0 +1,19 @@
+<p class="govuk-body">
+  <% if @planning_application.cancellable? %>
```

Comment:
This is a tiny thing, but what do you think about creating a new method for something like this? Something like `open?`, `active?` or `in_progress?`. I realise the method would probably just be `!cancellable?`, but it feels a little bit more explanatory in the logic here?

Reply:
Great idea! Will add to this PR.